manifests/pipelines/aiedge-e2e.pipeline.yaml

```diff
@@ -209,7 +224,7 @@ spec:
     - name: DOCKERFILE
       value: containerfile_repo/$(params.containerfileRelativePath)
     - name: CONTEXT
-      value: model_dir-$(params.model-name)/$(params.modelRelativePath)/$(params.model-name)
+      value: model_dir-$(params.model-name)/$(params.model-name)
```

Comment:
Also in that case change this decription:
```yaml
- name: model-dir
  type: string
  description: Directory below <modelRelativePath>/<model-name>/ to be used as MODEL_DIR in image build.
```
because we no longer have this directory structure. We eliminated the modelRelativePath.